wiki/Beatmap_Editor/Menu/en.md

```diff
@@ -1,441 +1,256 @@
-<tbody>
-<tr class="odd">
-<td><p>colspan=2| Redo/Undo commands</p></td>
-</tr>
-<tr class="even">
-<td><p>colspan=2|Do note that these commands can be disabled at <a href="Options#Editor_section" title="wikilink">Options</a> sidebar to prevent crash when editing marathon songs.</p></td>
-</tr>
-<tr class="odd">
-<td><p>Undo<br />
-(Ctrl + Z)</p></td>
-</tr>
-<tr class="even">
-<td><p>Redo<br />
-(Ctrl + Y)</p></td>
-</tr>
-<tr class="odd">
-<td><p>colspan=2| Edit commands (Select a note before use)</p></td>
-</tr>
-<tr class="even">
-<td><p>Cut<br />
-(Ctrl + X)</p></td>
-</tr>
-<tr class="odd">
-<td><p>Copy<br />
-(Ctrl + C)</p></td>
-</tr>
-<tr class="even">
-<td><p>Paste<br />
-(Ctrl + V)</p></td>
-</tr>
-<tr class="odd">
-<td><p>Delete<br />
-(Del / Delete)</p></td>
-</tr>
-<tr class="even">
-<td><p>colspan=2| Special commands</p></td>
-</tr>
-<tr class="odd">
-<td><p>Select All<br />
-(Ctrl + A)</p></td>
-</tr>
-<tr class="even">
-<td><p>Clone<br />
-(Ctrl + D)</p></td>
-</tr>
-<tr class="odd">
-<td><p>colspan=2| Rotation commands</p></td>
-</tr>
-<tr class="even">
-<td><p>Reverse selection<br />
-(Ctrl + G)</p></td>
-</tr>
-<tr class="odd">
-<td><p>Flip Horizontally<br />
-(Ctrl + H)</p></td>
-</tr>
-<tr class="even">
-<td><p>Flip Vertically<br />
-(Ctrl + J)</p></td>
-</tr>
-<tr class="odd">
-<td><p>Rotate 90° Clockwise<br />
-(Ctrl + &gt;)</p></td>
-</tr>
-<tr class="even">
-<td><p>Rotate 90° Anticlockwise<br />
-(Ctrl + &lt;)</p></td>
-</tr>
-<tr class="odd">
-<td><p>Rotate by...<br />
-(Ctrl + Shift + R)</p></td>
-</tr>
-<tr class="even">
-<td><p>Scale by...<br />
-(Ctrl + Shift + S)</p></td>
-</tr>
-<tr class="odd">
-<td><p>colspan=2| Reset Buttons</p></td>
-</tr>
-<tr class="even">
-<td><p>Reset selected objects' samples</p></td>
-</tr>
-<tr class="odd">
-<td><p>Reset all samples</p></td>
-</tr>
-<tr class="even">
-<td><p>Reset all combo colour</p></td>
-</tr>
-<tr class="odd">
-<td><p>Reset breaks</p></td>
-</tr>
-<tr class="even">
-<td><p>colspan=2| Nudge commands</p></td>
-</tr>
-<tr class="odd">
-<td><p>Nudge Backward<br />
-(J)</p></td>
-</tr>
-<tr class="even">
-<td><p>Nudge Forward<br />
-(K)</p></td>
-</tr>
-</tbody>
-</table>
+-----
+
+![Edit menu](M_Edit.jpg "Edit menu")
+
+Edit commands. Same as default MS shortcuts.
+
+### Redo/Undo commands
+
+Do note that these commands can be disabled at [Options](/wiki/Options) sidebar to prevent crash when editing marathon songs.
+
+| Name | Description |
+| ---- | ----------- |
+| Undo (Ctrl + Z) | Undo one step. |
+| Redo (Ctrl + Y) | Redo one step. |
+
+### Edit commands (Select a note before use)
+
+| Name | Description |
+| ---- | ----------- |
+| Cut (Ctrl + X) | Cut the note(s). |
+| Copy (Ctrl + C) | Copy the note(s) |
+| Paste (Ctrl + V) | Paste the note(s) |
+| Delete (Del / Delete) | Delete the note(s) |
+
+### Special commands
+
+| Name | Description |
+| ---- | ----------- |
+| Select All (Ctrl + A) | Select every single notes. |
+| Clone (Ctrl + D) | Copy + Paste. |
```

Comment:
change `Ctrl + D` to `` `Ctrl` + `D` ``
could also note that doing this will paste the notes 1(?) whole measure over